src/synonyms.ts

```diff
@@ -0,0 +1,92 @@
+import * as _request from 'request-promise';
+import * as fs from 'fs';
+import * as Ajv from 'ajv';
+import { request } from 'https';
+
+import * as R from 'ramda';
+
+
+export async function synonyms({
+   payload }: { payload: { symbol: string } }) {
+
+  const data: any = payload;
+  const symbol: string = payload.symbol;
+  const url_symbol = `https://rest.genenames.org/fetch/symbol/${symbol}`;
+
+  const options_symbol = {
+    method: 'GET',
+    uri: url_symbol as string,
+    json: true
+    };
+
+  const result: any = await _request(options_symbol)
+    .then((res: any) => {
+      if (typeof res.response.docs[0] !== 'undefined' && res.response.docs.length === 1 ) {
```

Comment:
This blows up if docs is undefined:
`R.isEmpty(R.path(['x', 'docs'], o))`

Reply:
Done.